automation/check-patch.e2e-multus-functests.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+set -xeuE
+
+# This script should be able to execute multus
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# Usage:
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-multus-functests.sh
+
+teardown() {
+    rm -rf "${TMP_COMPONENT_PATH}"
+    make cluster-down
+}
+
+function __get-tools() {
+    mkdir -p bin
+
+    JQ_PATH=./bin/jq
```

Comment:
Should not we make this into a absolute path under TMP_PROJECT_PATH?. This seems that the helper function would just create a bin directory somewhere and then we need to figure out where.

Could you think about a way to share the path with the caller? Or the caller could pass the path where to install the helper.

Well, what I mind is that it creates **something somewhere**.

And who adds JQ_PATH under the PATH?

Reply:
it's supposed to be inside the multus repo: `${TMP_COMPONENT_PATH}/e2e`

>And who adds JQ_PATH under the PATH?

[well](https://github.com/intel/multus-cni/blob/master/e2e/get_tools.sh)... :)


anyway I'll change it  to be absolute for better readability